### PR TITLE
Disable invites step in wizard when local_logins are turned off

### PR DESCRIPTION
**Pull request by pmusaraj**

### CODE REVIEW

**Comment by jjaffeux** [comment]
LGTM 👍

File: app/serializers/wizard_step_serializer.rb [hunk 1]
```diff
@@ -39,6 +39,8 @@ def translate(sub_key, vars = nil)
   end
 
   def description
+    return translate("disabled") if object.disabled
```
Comment (tgxworld) [inline app/serializers/wizard_step_serializer.rb]:
```suggestion
    translate(object.disabled ? "disabled" : translate("description", base_path: Discourse.base_path))
```

File: app/serializers/wizard_step_serializer.rb [hunk 2]
```diff
@@ -39,6 +39,8 @@ def translate(sub_key, vars = nil)
...
+
     translate("description", base_path: Discourse.base_path)
```
Comment (tgxworld) [inline app/serializers/wizard_step_serializer.rb]: Not related but I don't think we need `base_path` to be passed here.

File: spec/components/wizard/wizard_builder_spec.rb [hunk 3]
```diff
@@ -27,4 +27,14 @@
     expect(wizard.steps).to be_blank
   end
 
+  it "returns wizard with disabled invites step when local_logins are off" do
```
Comment (tgxworld) [inline spec/components/wizard/wizard_builder_spec.rb]: I think we should test that the right translation is being displayed as well.

**Comment by jjaffeux** [comment]
@pmusaraj if you have some time can you merge it when you start your day please? I need it for other works. Thanks ❤️

**Comment by tgxworld** [comment]
@jjaffeux feel free to merge it now. My comments are minor and can be addressed later :)

**Pull request merged**